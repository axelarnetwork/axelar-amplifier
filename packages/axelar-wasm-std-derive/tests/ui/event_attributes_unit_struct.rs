use axelar_wasm_std_derive::EventAttributes;

#[derive(EventAttributes)]
struct UnitStruct;

fn main() {}
