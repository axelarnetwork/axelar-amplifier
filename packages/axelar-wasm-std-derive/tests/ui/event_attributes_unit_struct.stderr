error: EventAttributes can only be derived for named structs, not unit structs
 --> tests/ui/event_attributes_unit_struct.rs:4:8
  |
4 | struct UnitStruct;
  |        ^^^^^^^^^^
